RELEASE-NOTES.txt

```diff
@@ -1,7 +1,14 @@
+1.32.0
+------
+* [***] Adds Copy, Cut, Paste, and Duplicate functionality to blocks
+* [***] Add support for mentions.
```

Comment:
> Add support for mentions

Should we put this in the release notes if this will only be available internally? I think we wouldn't want this to occur in App Store, right?

Reply:
Good point, we definitely don't want this in the release notes for the 15.3 release notes. Although, at the gutenberg-mobile level, the mentions capability is added in this release--we're just turning that capability off in release builds for the main apps. So I'm not really sure what the right approach is here. If the gutenberg-mobile release-notes were really like a library's release notes, I would say it should be included, but we've always kind of used these release notes to just note things that will be user-facing in the main apps, and mentions will not be user-facing in the main apps release. Basically, I don't feel strongly either way. Any thoughts @SergioEstevao ?

Reply:
Wait, I thought for a minute I am commenting under a WPiOS PR, my apologies for the confusion. I think it is fine to keep it in gb-mobile level.